### PR TITLE
[INFRA] Bump google{test,benchmark}

### PR DESCRIPTION
**Pull request by eseiler**

### CODE REVIEW

File: test/unit/test/pretty_printing_test.cpp [hunk 1]
```diff
@@ -168,10 +154,8 @@ inline debug_stream_type<char_t> & operator<<(debug_stream_type<char_t> & s, you
 
 } // namespace seqan3
 
-TEST(pretty_printing, seqan3_detail_mixed_seqan3_output)
+TEST(pretty_printing, seqan3_detail_inherit)
 {
-    // seqan3 types should always produce the same result
```
Comment (smehringer) [inline test/unit/test/pretty_printing_test.cpp:173]: why remove the comment?

File: test/unit/test/pretty_printing_test.cpp [hunk 2]
```diff
@@ -37,78 +34,68 @@ auto debug_str = [](auto && v)
...
-#if GTEST_INTERNAL_HAS_OPTIONAL // googletest at head (>=1.10.x)
+TEST(pretty_printing, optional)
+{
     EXPECT_EQ(gtest_str(std::optional<int>{}), "(nullopt)"s);
-#else // googletest <= 1.10.0
-    EXPECT_EQ(gtest_str(std::optional<int>{}), "<VALUELESS_OPTIONAL>"s);
-#endif
     EXPECT_EQ(debug_str(std::optional<int>{}), "<VALUELESS_OPTIONAL>"s);
+
+    EXPECT_EQ(gtest_str(std::nullopt), "(nullopt)"s);
```
Comment (smehringer) [inline test/unit/test/pretty_printing_test.cpp:60]:
```suggestion
```
This is already tested above, no?

File: test/unit/test/pretty_printing_test.cpp [hunk 3]
```diff
@@ -37,78 +34,68 @@ auto debug_str = [](auto && v)
...
-    EXPECT_EQ(dna_tuple1, dna_tuple2); // change value to test
+    auto dna_tuple = std::make_tuple('A'_dna4, 'C'_dna4);
+    EXPECT_EQ(gtest_str(dna_tuple), "(A, C)"s);
+    EXPECT_EQ(debug_str(dna_tuple), "(A,C)"s);
 
-    auto dna_sequence_tuple1 = std::make_tuple("AC"_dna4, "GT"_dna4);
-    auto dna_sequence_tuple2 = std::make_tuple("AC"_dna4, "GT"_dna4);
-    EXPECT_EQ(gtest_str(dna_sequence_tuple1), "(AC, GT)"s);
-    EXPECT_EQ(debug_str(dna_sequence_tuple1), "(AC,GT)"s);
-    EXPECT_EQ(dna_sequence_tuple1, dna_sequence_tuple2); // change value to test
```
Comment (smehringer) [inline test/unit/test/pretty_printing_test.cpp:111]: you always removed the "change value to test". Why? seemed helpful for testing, no?

File: test/unit/test/pretty_printing_test.cpp (hunk 2 above)
Comment (eseiler) [reply test/unit/test/pretty_printing_test.cpp:60]: Where? The other line uses `std::optional<int>{}`

File: test/unit/test/pretty_printing_test.cpp (hunk 1 above)
Comment (eseiler) [reply test/unit/test/pretty_printing_test.cpp:173]: Re-added :)

**Review by SGSSGene — APPROVED** [review]

File: test/unit/test/pretty_printing_test.cpp (hunk 2 above)
Comment (smehringer) [reply test/unit/test/pretty_printing_test.cpp:60]: true. ops

File: test/unit/test/pretty_printing_test.cpp (hunk 3 above)
Comment (smehringer) [reply test/unit/test/pretty_printing_test.cpp:111]: I think you could alter one of the tuples and than the EXPECT_EQ will fail which triggers the output of gtest so you can adapt the gtest output?

But now that you removed it it's probably not worth the hassle to reintroduce.

**Review by smehringer — COMMENTED** [review]
see inline comments

File: test/unit/test/pretty_printing_test.cpp (hunk 3 above)
Comment (eseiler) [reply test/unit/test/pretty_printing_test.cpp:111]: Ah, that wasn't clear at all from the comment :D

But if gtest changes, the `gtest_str` also changes and prints the expected/actual gtest output

Comment (smehringer) [reply test/unit/test/pretty_printing_test.cpp:111]: yes propbably. You can leave the PR as is FMPOV

**Review by smehringer — APPROVED** [review]

**Pull request merged**